### PR TITLE
disable 2 tests

### PR DESCRIPTION
**Pull request by ranasaria**
Disabling 2 tests temporarily. Tracking using issue: #12529 12529

### CODE REVIEW

File: extensions/azdata/src/test/azdata.test.ts [hunk 1]
```diff
@@ -102,7 +100,7 @@ describe('azdata', function () {
 			sinon.stub(vscode.window, 'showInformationMessage').returns(Promise.resolve(<any>loc.yes));
 		});
 
-		it('successful update', async function (): Promise<void> {
+		it.skip('successful update', async function (): Promise<void> {
```
Comment (ranasaria) [inline extensions/azdata/src/test/azdata.test.ts:103]: add a workitem for skipped tests for re-enablment

**Comment by Charles-Gagnon** [comment]
Are you seeing these tests failing yourself? I'm seeing different ones fail - but I'm on windows which may be why.

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
Do we need this? The tests pass for me on windows and on Linux in the labs, is this an Mac specific issue?

**Comment by ranasaria** [comment]
No I saw these fail on windows. But if they pass for you then I will abandon skipping them but just investigate to see why they were failing for me on my windows machine.

**Pull request closed without merging**